Projections/Windows/Windows.csproj

```diff
@@ -1,74 +1,35 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
-    <Platforms>x64;x86</Platforms>
-    <LangVersion>preview</LangVersion>
```

Comment:
What was this used for/why don't we need it now? I see in other files (UnitTest), it is different (8).

Reply:
just set it to preview in a central location (directory.build.props)